assets/app/views/_project-page.html

```diff
@@ -16,10 +16,8 @@
     <div class="container-fluid">
       <div class="sidebar-header sidebar-header-bleed-left sidebar-header-bleed-right">
         <div class="actions pull-right">
-          <!-- If we have any header actions for the sidebar in the future, they would go here
-              and be links like below
-              <a href="#">Example</a>
-          -->
+	  <a href="#" ng-click="overviewToggle()" ng-if="overviewMode == 'tiles'">Topology</a>
```

Comment:
@jwforres @abomas we have a couple of options which I can add links to later today.  I do like the link which you added above, although not sure if that would be identifiable at the smallest size.  Are you still using SVG format?